recipes/liburing/all/conandata.yml

```diff
@@ -1,7 +1,51 @@
 sources:
+  "2.1":
+    url: "https://github.com/axboe/liburing/archive/liburing-2.1.tar.gz"
+    sha256: "f1e0500cb3934b0b61c5020c3999a973c9c93b618faff1eba75aadc95bb03e07"
   "2.0":
     url: "https://github.com/axboe/liburing/archive/liburing-2.0.tar.gz"
     sha256: "ca069ecc4aa1baf1031bd772e4e97f7e26dfb6bb733d79f70159589b22ab4dc0"
   "0.7":
     url: "https://github.com/axboe/liburing/archive/liburing-0.7.tar.gz"
     sha256: "8e2842cfe947f3a443af301bdd6d034455536c38a455c7a700d0c1ad165a7543"
+  # "0.6":
+  #   url: "https://github.com/axboe/liburing/archive/liburing-0.6.tar.gz"
+  #   sha256: "cf718a0a60c3a54da7ec82a0ca639a8e55d683f931b9aba9da603b849db185de"
+  # "0.5":
+  #   url: "https://github.com/axboe/liburing/archive/liburing-0.5.tar.gz"
+  #   sha256: "597bcbea647e8d9f9e8242a445cd1a68b57a031bda1fa7d8e550faffd71f6b23"
+  # "0.4":
+  #   url: "https://github.com/axboe/liburing/archive/liburing-0.4.tar.gz"
+  #   sha256: "828c7df580a41ce197ee9e8f7f4b6620cd868bb19c9ba33c7a464533213bad38"
+  # "0.3":
+  #   url: "https://github.com/axboe/liburing/archive/liburing-0.3.tar.gz"
+  #   sha256: "b936144093fc999f2affc35baee1dbdf18bffb04eb96f20680223edf90fc406d"
+  # "0.2":
+  #   url: "https://github.com/axboe/liburing/archive/liburing-0.2.tar.gz"
+  #   sha256: "f067f08d73c034221332d5538f428c2d00eda45cb81b3ae79ccff1550baa6990"
+  # "0.1":
+  #   url: "https://github.com/axboe/liburing/archive/liburing-0.1.tar.gz"
+  #   sha256: "6deecf4d2ebe677cea00d650fe90b8e10d8aeffb0f4b12ef2315f0f3a8e9724a"
```

Comment:
```suggestion
```

Don't keep old versions. Exposing it, you are giving the idea that all those versions are still "maintained" or were supported by Conan, which is not true.